java-shared-dependencies/unmanaged-dependency-check/src/main/java/com/google/cloud/UnmanagedDependencyCheck.java

```diff
@@ -0,0 +1,78 @@
+package com.google.cloud;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.google.cloud.tools.opensource.classpath.ClassPathBuilder;
+import com.google.cloud.tools.opensource.classpath.DependencyMediation;
+import com.google.cloud.tools.opensource.dependencies.Bom;
+import com.google.cloud.tools.opensource.dependencies.MavenRepositoryException;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.eclipse.aether.resolution.ArtifactDescriptorException;
+import org.eclipse.aether.version.InvalidVersionSpecificationException;
+
+public class UnmanagedDependencyCheck {
+
+  public static void main(String[] args)
```

Comment:
Would you add Javadoc for the arguments?

Reply:
done.

Reply:
I think passing the version of Java shared dependencies will be come inconvenient.

Reply:
Users of unmanaged dependency check don't need to pass this parameter.

The version of shared dependencies needs update every two weeks so I think it's fine to set a parameter.